.terraform/terraform.tfstate

```diff
@@ -0,0 +1,29 @@
+{
+    "version": 3,
+    "serial": 1,
+    "lineage": "abf406aa-985a-d96f-7477-f5a782d01696",
+    "backend": {
+        "type": "cloud",
+        "config": {
+            "hostname": null,
+            "organization": "hugodiniz",
+            "token": null,
```

Comment:
After we can use this token for authentication in pipeline, right?

Reply:
This is only the terraform state file. There's a copy of this in the cloud, so I believe I can remove this sometime soon